src/utils/global/v1Utils.ts

```diff
@@ -0,0 +1,8 @@
+export function getSubDomain(url: string): string {
+    if (!url) {
+        return url
```

Comment:
Maybe nice to utilize the ?. 

This could clean it up to:
`return url?.replace("http://", "").replace("https://", "").split(":")[0];`